packages/block-library/src/navigation/index.php

```diff
@@ -537,7 +539,7 @@ private static function get_responsive_container_markup( $attributes, $inner_blo
 			$toggle_aria_label_close,
 			esc_attr( implode( ' ', $responsive_container_classes ) ),
 			esc_attr( implode( ' ', $open_button_classes ) ),
-			esc_attr( safecss_filter_attr( $colors['overlay_inline_styles'] ) ),
+			( ! empty( $overlay_inline_styles ) ) ? "style=$overlay_inline_styles" : '',
```

Comment:
The PR isn't working at the moment, this is just missing some quotes around the style:
```suggestion
			( ! empty( $overlay_inline_styles ) ) ? "style=\"$overlay_inline_styles\"" : '',
```

To reproduce:
1. Set a navigation block to Overlay - 'Always'
2. Add some custom colors for the overlay menu
3. Save and view the frontend - observe the custom colors aren't applied.

Reply:
@talldan added missing quotes.
Thanks for suggestion.